storage/bootstrap/result/options.go

```diff
@@ -32,21 +33,25 @@ const (
 )
 
 type options struct {
-	clockOpts         clock.Options
-	instrumentOpts    instrument.Options
-	blockOpts         block.Options
-	newBlocksLen      int
-	seriesCachePolicy series.CachePolicy
+	clockOpts               clock.Options
+	instrumentOpts          instrument.Options
+	blockOpts               block.Options
+	newBlocksLen            int
+	seriesCachePolicy       series.CachePolicy
+	mutableSegmentAllocator index.MutableSegmentAllocator
 }
 
 // NewOptions creates new bootstrap options
 func NewOptions() Options {
+	iopts := instrument.NewOptions()
 	return &options{
 		clockOpts:         clock.NewOptions(),
-		instrumentOpts:    instrument.NewOptions(),
+		instrumentOpts:    iopts,
 		blockOpts:         block.NewOptions(),
 		newBlocksLen:      defaultNewBlocksLen,
 		seriesCachePolicy: series.DefaultCachePolicy,
+		mutableSegmentAllocator: index.NewDefaultMutableSegmentAllocator(index.NewOptions().
```

Comment:
can probably skip overriding `InstrumentOptions()`? they're both going to end up with the default `instrument.NewOptions()`

Reply:
Sure thing.